purchase_triple_discount/models/triple_discount_mixin.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019 Tecnativa - David Vidal
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import functools
+
+from odoo import api, fields, models
+
+
+class TripleDiscount(models.AbstractModel):
```

Comment:
this mixin should be in account_invoice_triple_discount module. don't you think ?

Reply:
Maybe, we have to look in detail to make sure it is compatible with both modules.